generated/js/src/porter/v1/cluster_control_plane_pb.d.ts

```diff
@@ -13,6 +13,7 @@ import type { Contract, ContractRevision } from "./contract_pb.js";
 import type { ContractComplianceCheckGroup, EnumComplianceProfile, EnumComplianceVendor, VendorComplianceCheck } from "./compliance_pb.js";
 import type { AppImage, Build, Deletions, DeploymentTarget, DeploymentTargetIdentifier, EnumAppRevisionStatus, EnvGroup, EnvGroupVariables, EnvVariableDeletions, JobRun, PorterApp } from "./porter_app_pb.js";
 import type { Addon, PrerequisiteAddon } from "./addons_pb.js";
+import type { AppEventType } from "./agent_app_event_types_pb.js";
```

Comment:
nit: would rename `agent_app_event_types.proto` to `app_events.proto`, first because we might add more messages to this file that aren't just types, and second because this should be abstracted from whatever particular source the events come from (in this case agent)